examples/scraper_example.py

```diff
@@ -3,18 +3,18 @@
 """
 
```

Comment:
We should avoid consistently making changes to this example file. Can you just set the standard examples to citalopram?